pkg/process/checks/net.go

```diff
@@ -194,3 +230,25 @@ func getCtrIDsByPIDs(pids []int32) map[int32]string {
 	}
 	return Process.filterCtrIDsByPIDs(pids)
 }
+
+// countConnectionsPerPID counts the number of connections by type by PID
+func countConnectionsPerPID(conns []*model.Connection) map[int32]ConnectionsCounts {
+	res := map[int32]ConnectionsCounts{}
```

Comment:
Does this function need to use lock?

Reply:
It's not linked to the check so It should be fine 👍